src/reanimated2/__mocks__/MutableValue.js

```diff
@@ -0,0 +1,16 @@
+export default class MutableValue {
+  static MUTABLE_ID = 1;
+
+  constructor(value) {
+    this._id = MutableValue.MUTABLE_ID++;
```

Comment:
Do we need ids for this? It's not used anywhere.

Reply:
No, we don't. Removing...